### ISSUE TITLE
Need to sync repo with the origin book

### ISSUE THREAD

**Issue by Erfankam** [pomokhtari/rust-book-fa#7]
@pouriamokhtari  Since this repo is diverged from the origin for a long time and the origin has a lot of changes, I think it s better to resync repo with newer version of book. Is not it?

**Comment by p0m0h3** [comment]
Makes sense. Problem is translations need to be copied over to the `FARSI` folder as well. Will try to do it.

**Comment by p0m0h3** [comment]
41fab511eec753f6757efd614f118a05a9024afa merges with upstream.

**Comment by Erfankam** [comment]
I think we can mark this issue as closed or done. Thanks again.

**Comment by p0m0h3** [comment]
I don't think it's closed... we need to sync the FARSI Directory (the existing translations with the new content of the book)

**Comment by Erfankam** [comment]
> sync the FARSI Directory

No, we do not need. It s needed just whenever a new version of book or new update is released. Now, the book the English part of the book is synced.

**Comment by Erfankam** [comment]
I think you mean resync old Farsi translations from new origin English version. You are right dude.
